splitwise/__init__.py

```diff
@@ -261,3 +263,20 @@ def createExpense(self,expense):
             expense = Expense(content["expenses"][0])
 
         return expense
+    
+    def createGroup(self, name=None, group_type=None, country_code=None):
+        # create expense group
+        group_info = dict(name=name, group_type=group_type, country_code=country_code)
+        content = self.__makeRequest(Splitwise.CREATE_GROUP_URL, "POST", group_info)
+        content = json.loads(content.decode("utf-8"))
+
+        return content
+
+    def addUserToGroup(self, first_name=None, last_name=None, email=None, id=0):
```

Comment:
It should take a user object as input. See createExpense.

id should be group_id for clarity